.github/workflows/ci.yml

```diff
@@ -24,18 +24,43 @@ jobs:
 
     runs-on: "ubuntu-latest"
 
+    strategy:
+      matrix:
+        php-version:
+          - "7.3"
```

Comment:
Instead of using the matrix here, I could also inline the values.

Would you prefer these values to be inlined?

Reply:
Static analysis is only done once, with one PHP version. If we do not need a matrix for that: great.

Reply:
👍